app/views/trains/edit.html.slim

```diff
@@ -1,3 +1,3 @@
 h1 Editing Train
-= render partial: 'form', locals: { url: train_path, method: :patch }
+= render partial: 'form', locals: { checked: train["active"], button: "Update", url: train_path, method: :patch }
```

Comment:
whats the point of the `checked` variable here?  can you not do the logic inside the html with just `train["active"]`? this seems to add complexity with no value?

Reply:
can we have a more descriptive variable name instead of `button` its not instantly apparent what that means.
We could use something that conveys its usage like `submit_type` or `submit_text` or something that signifies this is related to the submit buttons label